packages/grafana-ui/src/types/completion.ts

```diff
@@ -57,10 +61,15 @@ export interface CompletionItem {
   documentation?: string;
 
   /**
-   * A string that should be used when comparing this item
-   * with other items. When `falsy` the `label` is used.
+   * A string or number that should be used when comparing this
+   * item with other items. When `undefined` then `label` is used.
+   */
+  sortValue?: string | number;
```

Comment:
The same as with `prefixMatch`

Reply:
Marked `sortText` as `@depracted` and `sortValue` as`@alpha`